Pipfile

```diff
@@ -49,6 +49,8 @@ snapshottest = "*"
 gql = "*"
 pyhumps = "*"
 django-graphql-jwt = "*"
+djangorestframework = "*"
```

Comment:
Just curious, since DRF is already a dependency of wagtail I would've expected you to be able to work with it without adding it as a direct dependency, was that not the case?

Reply:
I just tried running it without using it as a direct dependency. And it does not work.
`ModuleNotFoundError: No module named 'rest_framework'`